test/test_models.py

```diff
@@ -697,6 +698,11 @@ def test_classification_model(model_fn, dev):
 
     _check_input_backprop(model, x)
 
+    # Clearing from memory explicitly
+    del model, x, out
+    gc.collect()
+    torch.cuda.empty_cache()
```

Comment:
Note: I tried putting this functionality into a function in `common_utils.py` but I realized that calling `del` inside another function doesn't really works. For illustration look at the following script:
```
def mydelete(xs):
  for x in xs:
    del x

obj_1 = "hello"
obj_2 = [1,2,3]

mydelete([obj_1, obj_2])
print(obj_1, obj_2) # This still works and the objects is not deleted properly!

del obj_1, obj_2
print(obj_1, obj_2) # This will trigger error
```